kubernetes/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
```

Comment:
Nice! So looks like you're using a multiphase Docker file - this is exactly the right way to build the controller.